Chummer/UI/Skills/SkillControl2.cs

```diff
@@ -568,5 +568,23 @@ private void SkillControl2_DpiChangedAfterParent(object sender, EventArgs e)
                 lblModifiedRating.MinimumSize = new Size((int) (50 * g.DpiX / 96.0f), 0);
             }
         }
+
+
+        /// <summary>
+        /// Refreshes the Tooltip and Displayed Dice Pool. Can be used in another Thread
+        /// </summary>
+        private void SkillControl2_RefreshPoolTooltipAndDisplay()
+        {
+            //TODO: Run this async somehow, QueueThreadSafe maybe.
+            if (_blnLoading || lblModifiedRating.Text != 0.ToString(GlobalOptions.CultureInfo))
```

Comment:
This seems like an odd set of conditions. Are you sure that this code is how you want this entire method to work?

Reply:
The intention was, to only update the text, when it is necessary and for some reason I missed the obvious stupid bug I would have implemented...